### ISSUE TITLE
🐞 New Customizer - tabbing ignores left sidebar

### ISSUE THREAD

**Issue by Guido07111975** [ClassicPress/ClassicPress#2461]
For clarity I will create a topic for each issue I encounter with the updated Customizer in Nightly.
Did report some of the issues in the Customizer PR already, but that one is merged and closed.

Tested with the Twenty Seventeen theme.

### Current behavior
While tabbing the Customizer and clicking the "Add a widget" button, the `widgets-left` sidebar opens and the selector (focus) should jump to the search bar there. Currently it does not. It seems to ignore whole sidebar.

(same applies to the "Add items" button of the Menu panel)

<img width="574" height="272" alt="image" src="https://github.com/user-attachments/assets/c2d28df7-d0a4-45e4-a596-228677b7d7ad" />

### ClassicPress version
2.7.0+nightly.20260508

### PHP version
8.2

**Comment by KTS915** [comment]
Good spot, @Guido07111975 !

It's not hard to fix the specific issues you've pointed out, but here are two related questions for you. What do you think should happen if the user keeps tabbing through the newly-opened panel? At the moment, this takes them into the live preview window. Is this how it should be, or should it go back to the main Customizer panel and, if so, to which element? (In WP, it takes them into the browser console, which is clearly terrible.)

And what should happen if the user hits `Shift + tab` when in the search box in either newly-opened panel? At the moment, this takes them to the preview mobile icon (albeit after two tabs, which we can clearly improve on). Do you think this is the correct behavior?

**Comment by Guido07111975** [comment]
I'm not an expert on this, I just navigate through the Customizer and expecting a logical order :-)

> What do you think should happen if the user keeps tabbing through the newly-opened panel?

I would say it should jump back to the "Add a Widget" / "Add Items" button, so user is able to close sub-panel again and navigate further.
And from the footer navigation it should jump to the Preview. This should be the case in all panels.

> And what should happen if the user hits Shift + tab when in the search box in either newly-opened panel? 

For sub-panels Widgets and Menus idem: jump back to the button.
